tests/GravityForms/Field/DownloadUrlTest.php

```
@@ -0,0 +1,75 @@
+<?php
+
+namespace Gravity_Forms\Gravity_Forms\Settings\Fields {
+	if ( ! class_exists( __NAMESPACE__ . '\Text' ) ) {
+		/**
+		 * Minimal stand-in for Gravity Forms' settings Text field, which is not
+		 * available in the unit-test context. Mirrors the prop-copying constructor
+		 * the real base class uses.
+		 */
+		class Text {
+			public function __construct( $props = [], $settings = null ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$settings'. (undefined)

(UnusedFormalParameter)